tests/app/organisation/test_invite_rest.py

```diff
@@ -5,33 +5,33 @@
 from flask import current_app, json
 from freezegun import freeze_time
 
-from app.models import INVITE_PENDING, Notification
 from tests import create_admin_authorization_header
 from tests.app.db import create_invited_org_user
 
 
 @pytest.mark.parametrize(
-    "platform_admin, expected_invited_by", ((True, "The GOV.UK Notify team"), (False, "Test User"))
+    "platform_admin, expected_invited_by", ((True, "The GOV.UK Emergency Alerts team"), (False, "Test User"))
 )
 @pytest.mark.parametrize(
     "extra_args, expected_start_of_invite_url",
     [
-        ({}, "http://localhost:6012/organisation-invitation/"),
-        ({"invite_link_host": "https://www.example.com"}, "https://www.example.com/organisation-invitation/"),
+        ({}, "https://admin.development.emergency-alerts.service.gov.uk"),
```

Comment:
Hardcoded url.

Reply:
Hard-coded strings and variables tend to be acceptable in tests:

- When something breaks, the test output is more direct and shows you what the issue is, and having the string co-located at the top of the test reduces investigation time
- The pytest fixtures are stood up before the app config is read, so if we replaced this hard-coded string with a config variable we would see a "app context not found" or something similar